src/marqo/README.md

```diff
@@ -4,47 +4,88 @@ you want.
 
 See [here](https://github.com/marqo-ai/marqo/blob/mainline/CONTRIBUTING.md#unit-tests) for how to run unit tests.
 
-## Select an option (from A-E) to get set up. In most cases, Option A is recommended. 
+## Running Marqo locally (outside of docker) for development
 
-### Option A. Run the Marqo application locally (outside of docker):
+There are two ways to run Marqo locally (outside of docker) for development: Option A. through `uvicorn`, 
+Option B. through your IDE (e.g. PyCharm). 
+We highly recommend using the Option A, as it allows you to set breakpoints and debug Marqo. 
+Before running Marqo locally, you will need to do some preparations to set up Vespa locally.
 
-1. Have a running Marqo-OS instance available to use. You can spin up a local instance with the following command
-(if you are using an arm64 machine, replace `marqoai/marqo-os:0.0.3` with `marqoai/marqo-os:0.0.3-arm`):
+### Preparations
+
+1. Clone the Marqo Github repo and cd into it
 ```bash
-docker run --name marqo-os -id -p 9200:9200 -p 9600:9600 -e "discovery.type=single-node" marqoai/marqo-os:0.0.3
+git clone https://github.com/marqo-ai/marqo.git
+cd marqo
 ```
 
-2. Clone the github repo
+2. Install Marqo dependencies
+```bash
+pip install -r requirements.txt
 ```
-git clone https://github.com/marqo-ai/marqo.git
+
+3. Pull and run the Vespa docker image
+```bash
+docker run --detach --name vespa --hostname vespa-tutorial \
+  --publish 8080:8080 --publish 19071:19071 --publish 19092:19092 \
+  vespaengine/vespa:latest
 ```
-3. Install marqo dependencies
+
+4. Install the vespa command line interface (CLI)
+Check out the [Vespa CLI documentation](https://docs.vespa.ai/en/vespa-cli.html) for how to install the CLI.
```

Comment:
I felt really strong urge to skip this step as it looks not really meaningful when you are lazy, I think experience would be just a little bit better if we include that stroke for installation there and point it to the documentation as well.

Reply:
I update the readme so we use curl. ez and simple now